gradle/libs.versions.toml

```diff
@@ -42,7 +42,7 @@ compose-compiler = "1.5.0"
 compose-constraint = "1.0.1"
 compose-material3 = "1.1.0"
 compose-navigation = "2.6.0"
-compose-destinations = "1.9.40-beta"
+compose-destinations = "1.9.51"
```

Comment:
This one can be problematic, I found right versions of accompanist and compose navigation to work with the current compose-destination version without the need to update anything else, because since 1.9.50 if I remember correctly it requires to bump target sdk to 34 and android gradle plugin version 🙈

Reply:
I want to bump AGP, especially when modularizing the app because there are some things that help with AGP 8, and avoid having to fix stuff twice. Unfortunately this will require some time that I don't have at the moment.


We can close this PR and let dependabot bring other dependencies for now.